controllers/cluster-registration/suite_test.go

```diff
@@ -601,40 +605,25 @@ var _ = BeforeSuite(func() {
 	// Launch the compute-operator manager
 	go func() {
 		defer GinkgoRecover()
-		logf.Log.Info("build controller")
-		build := exec.Command("go",
-			"build",
-			"-o",
-			controllerExecutable,
-			"../../main.go")
-		err := build.Run()
-		Expect(err).To(BeNil())
-
 		logf.Log.Info("run controller")
 
 		os.Setenv("POD_NAME", "installer-pod")
 		os.Setenv("POD_NAMESPACE", controllerNamespace)
-		controllerManager = exec.Command(controllerExecutable,
-			"manager",
-			"--kubeconfig",
-			testEnvKubeconfigFile,
-			// "--v=6",
-		)
 
-		// Create io.writer for manager log
-		managerLogFile := os.Getenv("MANAGER_LOG")
-		if len(managerLogFile) == 0 {
-			controllerManager.Stdout = os.Stdout
-			controllerManager.Stderr = os.Stderr
-		} else {
-			os.MkdirAll(filepath.Dir(filepath.Clean(managerLogFile)), 0700)
-			f, err := os.OpenFile(filepath.Clean(managerLogFile), os.O_WRONLY|os.O_CREATE, 0600)
-			Expect(err).To(BeNil())
-			defer f.Close()
-			controllerManager.Stdout = f
-			controllerManager.Stderr = f
-		}
-		err = controllerManager.Start()
+		rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
Did you mean to leave all of this in here?

Reply:
yes because I call here the manager main and not the root main.go which is for the installer and manager and so I have to add some cmds from the root main.go into the suite_test in order to call the cmd.Execute()